lib/Service/Appointments/BookingService.php

```diff
@@ -114,11 +114,20 @@ public function confirmBooking(Booking $booking, AppointmentConfig $config): Boo
 		$calendar = $this->calendarWriter->write($config, $startObj, $booking->getDisplayName(), $booking->getEmail(), $booking->getDescription());
 		$booking->setConfirmed(true);
 		$this->bookingMapper->update($booking);
+
 		try {
 			$this->mailService->sendBookingInformationEmail($booking, $config, $calendar);
+			$this->mailService->sendOrganizerBookingInformationEmail($booking, $config, $calendar);
 		} catch (ServiceException $e) {
-			$this->logger->info('Could not send booking information email after confirmation by user ' . $booking->getEmail(), ['exception' => $e]);
+			$this->logger->info('Could not send booking emails after confirmation from user ' . $booking->getEmail(), ['exception' => $e]);
 		}
+
+		try {
+			$this->mailService->sendOrganizerBookingInformationNotification($booking, $config);
+		} catch (\InvalidArgumentException $e) {
+			$this->logger->info('Could not send booking information notification after confirmation by user ' . $booking->getEmail(), ['exception' => $e]);
```

Comment:
nit: make it a warning or error unless there is a good reason to silently ignore error here

Reply:
just using the same logic as above for the emails.

Reply:
please give me a chance to see comments that do not get addressed. leave these github review comments unresolved.

Reply:
it did get changed. alright.